openid-connect-server/src/main/java/org/mitre/openid/connect/config/JsonMessageSource.java

```diff
@@ -54,9 +54,13 @@
 
 	private Map<Locale, List<JsonObject>> languageMaps = new HashMap<>();
 
-	@Autowired
 	private ConfigurationPropertiesBean config;
 
+	@Autowired
```

Comment:
We use `@Autowired` on fields instead of constructors throughout the rest of the project and I'd like to keep this somewhat consistent unless there's good reason to change just this one and not the others.

Reply:
It's a lot easier to insert test/mock objects into an object if you use constructor injection. The unit test I wrote requires no Spring context because it just constructs the config object as a POJO and passes it in during object construction. If you stick with field injection then you are tied to relying on the framework to construct objects correctly.

Reply:
Other unit tests in the project use mock injection (with `@InjectMocks`) and it works fine.

Reply:
I just updated to keep field injection and inject using @InjectMocks. Let me know if that works.